eng/packaging.targets

```diff
@@ -256,4 +256,10 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ValidateAssemblyVersionsInRefPack" 
+          Condition="$(_AssemblyInTargetingPack) == 'true' and '$(PreReleaseVersionLabel)' != 'servicing'" 
```

Comment:
Why do we care about the `PreReleaseVersionLabel` being different than servicing? Shouldn't it be equal to servicing?